dezoomify.py

```diff
@@ -261,7 +264,7 @@ def untile_image(self, output_destination):
         joining_progressbar = None
         if progressbar:
             download_progressbar = progressbar.ProgressBar(
-                widgets=['Downloading tiles: ',
+                widgets=['Loading tiles: ',
```

Comment:
Why *loading* rather than *downloading* ? At this point, the tiles are being **down**loaded from the server to a temporary folder.